java/flight/flight-core/src/main/java/org/apache/arrow/flight/FlightService.java

```diff
@@ -147,11 +149,23 @@ private void onCancel() {
       }
     }
 
+    private void onReady() {
+      logger.debug("Stream is ready for new messages.");
```

Comment:
This is going to spam logs a lot - best not to add it unless it's really valuable.